src/modules/launcher/Wox.Core/Plugin/PluginInstaller.cs

```diff
@@ -74,8 +74,8 @@ internal static void Install(string path)
                     UnZip(path, newPluginPath, true);
                     Directory.Delete(tempFoler, true);
 
-                    //exsiting plugins may be has loaded by application,
-                    //if we try to delelte those kind of plugins, we will get a  error that indicate the
+                    //existing plugins may be has loaded by application,
```

Comment:
`//existing plugins could be loaded by the application`

Reply:
```suggestion
                    //existing plugins could be loaded by the application,
```